<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>676</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>980</width>
     <height>650</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Головна сторінка</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="title">
           <string>Фільтр</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>грн.</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QCheckBox" name="checkBox_36">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Німеччина</string>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QCheckBox" name="checkBox_37">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Subaru</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QCheckBox" name="checkBox_34">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Toyota</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="do_1">
             <property name="maximumSize">
              <size>
               <width>144</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QCheckBox" name="checkBox_46">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Audi</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkBox_35">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Mercedes-Benz</string>
             </property>
            </widget>
           </item>
           <item row="16" column="0">
            <widget class="QCheckBox" name="checkBox_45">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Велика Британія</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_42">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>BMW</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <widget class="QCheckBox" name="checkBox_38">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Японія</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QCheckBox" name="checkBox_43">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Land-Rover</string>
             </property>
            </widget>
           </item>
           <item row="15" column="0">
            <widget class="QCheckBox" name="checkBox_33">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Франція</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QCheckBox" name="checkBox_47">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Volvo</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0">
            <widget class="QCheckBox" name="checkBox_40">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Porsche</string>
             </property>
            </widget>
           </item>
           <item row="17" column="0">
            <widget class="QCheckBox" name="checkBox_41">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Швеція</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QCheckBox" name="checkBox_39">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Peugeot</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_12">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>Марка:</string>
             </property>
            </widget>
           </item>
           <item row="12" column="0">
            <widget class="QLabel" name="label_11">
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>14</pointsize>
               <italic>false</italic>
              </font>
             </property>
             <property name="text">
              <string>Країна виробник:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="vid">
             <property name="maximumSize">
              <size>
               <width>144</width>
               <height>25</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Times New Roman</family>
               <pointsize>12</pointsize>
               <italic>true</italic>
              </font>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>426</width>
            <height>611</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>426</width>
            <height>611</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="title">
           <string>  Пошук автомобілей</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>30</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::ClickFocus</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="styleSheet">
           <string notr="true">background-color:  rgb(255, 255, 255); font: italic 14pt &quot;Times New Roman&quot;;</string>
          </property>
          <property name="title">
           <string>Додаткова інформація</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Модель</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Комплектація Автомобіля</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Ціна</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Додати до замовлення</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Сторінка покупця</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QSpoiler" name="groupBox_5">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>Профіль</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>Зміна даних користувача</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButton_6">
             <property name="text">
              <string>Вихід з облікового запису</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="pushButton_7">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="7" column="0" colspan="3">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Витрата палива</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLineEdit" name="lineEdit_7">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Тип палива</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QLineEdit" name="lineEdit_10">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Ціна</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Об'єм двигуна</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Тип КПП</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Потужність</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Комплектація</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Колір</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Модель</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Кількість місць</string>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QLineEdit" name="lineEdit_9">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEdit_4">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEdit_5">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="lineEdit_8">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="QLineEdit" name="lineEdit_11">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="lineEdit_3">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="10" column="3">
            <widget class="QLineEdit" name="lineEdit_13"/>
           </item>
           <item row="9" column="2" colspan="2">
            <widget class="QLineEdit" name="lineEdit_12">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Кількість дверей</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="lineEdit_6">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="tableView_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Видалити з замовлення</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Замовити</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Рейтинг</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSpoiler</class>
   <extends>QGroupBox</extends>
   <header location="global">../kursovaya/qspoiler.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
